sql/moz-fx-data-shared-prod/telemetry_derived/cohort_daily_statistics_v2/query.sql

```diff
@@ -0,0 +1,200 @@
+-- Get all clients who had activity in the last 28 days and their bit patterns.
+WITH submission_date_activity AS (
+  SELECT
+    client_id,
+    days_seen_bits,
+    (days_visited_1_uri_bits & days_interacted_bits) AS days_seen_dau_bits,
+    DATE(submission_date) as activity_date
+  FROM
+    telemetry.clients_last_seen_v1 -- this might cause an issue because definition of first_seen_date in this table is different from clients_first_seen_v2
+  WHERE
+    submission_date = @activity_date
+  GROUP BY
+    client_id,
+    submission_date,
+    days_seen_bits,
+    (days_visited_1_uri_bits & days_interacted_bits)
+),
+-- Get all the cohorts that are still in range of the current day of activity (196 days)
+cohorts_in_range AS (
+  SELECT
+    client_id,
+    first_seen_date AS cohort_date,
+    DATE(@activity_date) AS activity_date,
+    app_build_id,
```

Comment:
Remove this field as well. Pretty big in terms of unique values. See: https://docs.google.com/spreadsheets/d/1s_rCwApytiask9a7rcIF-Mj5_4p7GVjW2dqH0dQkd70/edit#gid=0

Reply:
I've taken out document_id, dl_token, app_build_id columns, rename activity_date to submission_date, rename cohort_date to first_seen_date to match clients_first_seen_28_days_later as per conversation with George

Reply:
I've also changed the retention date to look back 106 days instead of 196